lumicks/pylake/force_calibration/detail/driving_input.py

```diff
@@ -98,3 +99,16 @@ def estimate_driving_input_parameters(
     amp = np.exp(p[2] - 0.25 * p[1] ** 2 / p[0] + 0.5 * np.log(-np.pi / p[0])) * delta_freq
 
     return amp, freq
+
+
+def response_power_density(psd_data, sample_rate, driving_frequency, num_windows, freq_window=50.0):
```

Comment:
I think `response_power_density` reads a bit unclear when used.
The docstring is pretty clear, but I can think of any good suggestions...

Reply:
`power_density_at_driving_input_frequency` ?